### PR TITLE
Merge upstream v4.9.1

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

File: src/plugins/mer/meremulatordevice.cpp [hunk 1]
```diff
@@ -197,6 +197,7 @@ MerEmulatorDevice::MerEmulatorDevice(Core::Id id)
     , m_cpuCount(0)
     , m_vdiCapacityMb(0)
 {
+    init();
```
Comment (vigejolla) [inline src/plugins/mer/meremulatordevice.cpp]: The upstream moved away from having separate init() in commit 54385ad67be075ac14d0edae5c451ed3d925edc8, maybe we should follow their example?

File: src/plugins/mer/merconnection.cpp [hunk 2]
```diff
@@ -80,22 +80,20 @@ class MerConnectionRemoteShutdownProcess : public QObject
     MerConnectionRemoteShutdownProcess(QObject *parent)
         : QObject(parent)
         , m_runner(0)
-        , m_connectionError(SshNoError)
+        , m_connectionErrorOccured()
```
Comment (vigejolla) [inline src/plugins/mer/merconnection.cpp]: Please initialize your member variables

File: src/plugins/mer/meremulatordevice.cpp (hunk 1 above)
Comment (martyone) [reply src/plugins/mer/meremulatordevice.cpp]: Added commit 98780c6cab3880c179c8f45d14b1c6733d1b9b99 but decided to left init() there to not break history (it is not immediately adjacent to the constructor)

File: src/plugins/mer/merconnection.cpp (hunk 2 above)
Comment (martyone) [reply src/plugins/mer/merconnection.cpp]: Fixed (typo)

**Comment by martyone** [comment]
Newly added commits aafe420f2a1f326d7cd71cbc3fcd72586ccc009e and a9e0a6648a7a70ac26b74113626d4289feb04935 fix adding emulator manually.

**Comment by vigejolla** [comment]
LGTM so far

**Pull request merged**